src/components/Loginpage.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import '../index.css';
+// 
+const Loginpage =() => {
+        return (
+        <div>
+            <form>
+
+                {/* <div className="sign-in"> 
+                <h3>Sign In</h3>
+                </div> */}
+            
+                <div className="form-group">
+                    <h3>Sign In</h3>
+
+                    <input type="email" className="form-control" placeholder=" Enter Email" /><br/>
+                
+                    <input type="password" className="form-control" placeholder=" Enter Password" /><br/>
+
+                    <div>
+                        <input type="checkbox" id="terms" name="terms" />
+                        <label for="terms">Remember Me</label> <br/>
+                    </div>
+
+                    <button type="submit" className="btn">Log In</button>
+                    <p className="forgot-password text-right">
```

Comment:
It would be cool to setup [Eslint](https://eslint.org/) and [Prettier](https://prettier.io/) for consistent spacing and coding styles.
And the `input` fields, you could make them reusable.
And you can also take out the commented out codes as you don't need them anymore 😄